hack/lib/common.sh

```diff
@@ -57,12 +57,29 @@ fi
 
 # Get runtime deployment files
 echo "Getting runtime deployment files"
-wget -O ${VIRTLET_DEPLOYMENT_FILES_DIR}/libvirt-qemu  ${VIRTLET_DEPLOYMENT_FILES_SRC}/apparmor/libvirt-qemu
-wget -O ${VIRTLET_DEPLOYMENT_FILES_DIR}/libvirtd  ${VIRTLET_DEPLOYMENT_FILES_SRC}/apparmor/libvirtd
-wget -O ${VIRTLET_DEPLOYMENT_FILES_DIR}/virtlet  ${VIRTLET_DEPLOYMENT_FILES_SRC}/apparmor/virtlet
-wget -O ${VIRTLET_DEPLOYMENT_FILES_DIR}/vms  ${VIRTLET_DEPLOYMENT_FILES_SRC}/apparmor/vms
-wget -O ${VIRTLET_DEPLOYMENT_FILES_DIR}/vmruntime.yaml  ${VIRTLET_DEPLOYMENT_FILES_SRC}/data/virtlet-ds.yaml
-wget -O ${VIRTLET_DEPLOYMENT_FILES_DIR}/images.yaml  ${VIRTLET_DEPLOYMENT_FILES_SRC}/images.yaml
+if [[ (${OVERWRITE_DEPLOYMENT_FILES} == "true") || (! -f ${VIRTLET_DEPLOYMENT_FILES_DIR}/libvirt-qemu) ]]; then
+  wget -O ${VIRTLET_DEPLOYMENT_FILES_DIR}/libvirt-qemu  ${VIRTLET_DEPLOYMENT_FILES_SRC}/apparmor/libvirt-qemu
+fi
+
+if [[ (${OVERWRITE_DEPLOYMENT_FILES} == "true") || (! -f ${VIRTLET_DEPLOYMENT_FILES_DIR}/libvirtd) ]]; then
+  wget -O ${VIRTLET_DEPLOYMENT_FILES_DIR}/libvirtd  ${VIRTLET_DEPLOYMENT_FILES_SRC}/apparmor/libvirtd
+fi
+
+if [[ (${OVERWRITE_DEPLOYMENT_FILES} == "true") || (! -f ${VIRTLET_DEPLOYMENT_FILES_DIR}/virtlet) ]]; then
+  wget -O ${VIRTLET_DEPLOYMENT_FILES_DIR}/virtlet  ${VIRTLET_DEPLOYMENT_FILES_SRC}/apparmor/virtlet
+fi
+
+if [[ (${OVERWRITE_DEPLOYMENT_FILES} == "true") || (! -f ${VIRTLET_DEPLOYMENT_FILES_DIR}/vms) ]]; then
+  wget -O ${VIRTLET_DEPLOYMENT_FILES_DIR}/vms  ${VIRTLET_DEPLOYMENT_FILES_SRC}/apparmor/vms
+fi
+
+if [[ (${OVERWRITE_DEPLOYMENT_FILES} == "true") || (! -f ${VIRTLET_DEPLOYMENT_FILES_DIR}/vmruntime.yaml) ]]; then
+  wget -O ${VIRTLET_DEPLOYMENT_FILES_DIR}/vmruntime.yaml  ${VIRTLET_DEPLOYMENT_FILES_SRC}/data/virtlet-ds.yaml
+fi
+
+if [[ (${OVERWRITE_DEPLOYMENT_FILES} == "true") || (! -f ${VIRTLET_DEPLOYMENT_FILES_DIR}/images.yaml) ]]; then
+  wget -O ${VIRTLET_DEPLOYMENT_FILES_DIR}/images.yaml  ${VIRTLET_DEPLOYMENT_FILES_SRC}/images.yaml
+fi
```

Comment:
maybe put these into a function to prevent repeating it

Reply:
good point. will do so.